runtime/src/main/java/io/micronaut/jackson/JacksonConfiguration.java

```diff
@@ -218,4 +227,15 @@ public void setSerializationInclusion(JsonInclude.Include serializationInclusion
             this.serializationInclusion = serializationInclusion;
         }
     }
+
+    /**
+     * Sets the property naming strategy.
+     *
+     * @return
+     */
+    public void setPropertyNamingStrategy(String propertyNamingStrategy) {
```

Comment:
The type here should be the jackson type. You can write a converter to convert the string to the type.

Reply:
I will add a converter.

Reply:
@jameskleeh Are you referring to creating an implementation of a io.micronaut.core.convert.TypeConverter?

Reply:
@msanssouci Yep!